paf-mvp-client-express/src/client-node.ts

```diff
@@ -61,129 +61,166 @@ export class ClientNode extends Node {
     // *****************************************************************************************************************
     // ************************************************************************************************************ REST
     // *****************************************************************************************************************
+    this.setEndpointConfig('GET', jsonProxyEndpoints.read, {
+      endPointName: 'Read',
+    });
     this.app.expressApp.get(
       jsonProxyEndpoints.read,
+      this.beginHandling,
       this.websiteIdentityValidator.cors,
       this.websiteIdentityValidator.checkOrigin,
-      this.startSpan(jsonProxyEndpoints.read),
       this.restBuildUrlToGetIdsAndPreferences,
-      this.catchErrors(jsonProxyEndpoints.read),
-      this.endSpan(jsonProxyEndpoints.read)
+      this.catchErrors,
+      this.endHandling
     );
 
+    this.setEndpointConfig('POST', jsonProxyEndpoints.write, {
+      endPointName: 'Write',
+    });
```

Comment:
Would it be more readable if we gather all the configurations in one place?

Example:
```typescript
const configs = [
{ 'POST', jsonProxyEndpoints.write,       { endPointName: 'Write' } },
{ 'GET', jsonProxyEndpoints.verify3PC, { endPointName: 'Verify3PC'} },
]
```